### DISCUSSION TITLE
How to calculate the PK parameters in Mobi?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1619] [How-Tos] [answered]
Hi! I wonder is there a way to calculate the PK parameters (AUC, Cmax, etc.) in Mobi directly? How can I get these parameters ?

Kind regards,

**Comment by wilbertdew** [comment, marked as answer]
@Gre-ux Directly in Mobi is not possible I believe, but you can save as a pkml file and import this in PK-Sim or use the R package to calculate the PK parameters

**Reply by Gre-ux** [reply]
Thanks very much for your reply! I have used R package to calculate.
